content/en/tracing/trace_explorer/trace_view.md

```diff
@@ -211,15 +211,15 @@ View [Profiles][1] to identify lines of code related to performance issues. The
 [1]: /profiler/connect_traces_and_profiles/#identify-code-hotspots-in-slow-traces
 
 {{% /tab %}}
-{{% tab "Span Links (Beta)" %}}
+{{% tab "Span Links" %}}
 
 <div class="alert alert-info">Span link support is in beta.</div>
```

Comment:
```suggestion
```
Guessing this should be removed as well?

Reply:
Yes!